frontend/plugins/insurance_ui/src/pages/insurance/VendorDetailPage.tsx

```
@@ -79,14 +81,14 @@ export const VendorDetailPage = () => {
   };
 
   const handleDelete = async (userId: string) => {
-    if (confirm('Are you sure you want to delete this user?')) {
+    if (confirm(t('confirm-delete-user'))) {
```

ESLint

[error] Unexpected use of 'confirm'.

(no-restricted-globals)